Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml

```diff
@@ -2229,7 +2228,629 @@ script:
     - contextPath: GSuite.Group.nonEditableAliases
       description: List of the group's non-editable alias email addresses that are outside of the account's primary domain or subdomains.
       type: String
-  dockerimage: demisto/googleapi-python3:1.0.0.89487
+  - arguments:
+      - description: 'One of the following: user email address, alias email address, or the unique user ID.'
+        name: user_key
+        required: true
+    description: Retreive a group's details given a group key.
+    name: gsuite-user-reset-password
+    outputs:
+      - contextPath: GSuite.User.id
+        description: The unique ID for the user.
+        type: String
+      - contextPath: GSuite.User.primaryEmail
+        description: The user's primary email address.
+        type: String
+      - contextPath: GSuite.User.firstName
+        description: The user's first name.
+        type: String
+      - contextPath: GSuite.User.lastName
+        description: The user's last name.
+        type: String
+      - contextPath: GSuite.User.customerId
+        description: The unique ID for the customer's G Suite account.
+        type: String
+      - contextPath: GSuite.User.gender
+        description: Gender.
+        type: String
+      - contextPath: GSuite.User.suspended
+        description: Indicates if the user is suspended.
+        type: Boolean
+      - contextPath: GSuite.User.notesValue
+        description: Contents of notes.
+        type: String
+      - contextPath: GSuite.User.notesContentType
+        description: Content type of notes.
+        type: String
+      - contextPath: GSuite.User.isAdmin
+        description: Indicates a user with super administrator privileges.
+        type: Boolean
+      - contextPath: GSuite.User.creationTime
+        description: The time the user's account was created.
+        type: Date
+      - contextPath: GSuite.User.phones.value
+        description: A human-readable phone number. It may be in any telephone number format.
+        type: String
+      - contextPath: GSuite.User.phones.type
+        description: The type of phone number.
+        type: String
+      - contextPath: GSuite.User.phones.primary
+        description: Indicates if this is the user's primary phone number.
+        type: Boolean
+      - contextPath: GSuite.User.phones.customType
+        description: If the value of type is custom, this property contains the custom type.
+        type: String
+      - contextPath: GSuite.User.addresses.type
+        description: The address type.
+        type: String
+      - contextPath: GSuite.User.addresses.customType
+        description: If the address type is custom, this property contains the custom value.
```

Comment:
The value in 2281 should be more clear. 
This one is clear - If the address type is customer, then it will contain the custom.

In line 2281 it's not clear, I changed it.